source/includes/common-mc-admin-config.rst

```diff
@@ -1022,3 +1022,101 @@ Ensure you have at least one other admin user, such as one with the :userpolicy:
 If you do not have another admin user, disabling the root account locks administrative access to the deployment.
 
 .. end-minio-root-api-access
+
+
+.. kafka audit settings
+
+.. start-minio-kafka-audit-logging-brokers-desc
+
+A comma-separated list of Kafka broker addresses:
+
+``brokers="https://kafka-1.example.net:9092,https://kafka-2.example.net:9092"``
```

Comment:
I guess put this in a fenced code block so it will scroll?

![screenshot showing long line overlapping the right sidebar](https://github.com/minio/docs/assets/95028/f5547b90-bfbf-4b61-a35f-58a5c794bd35)

Reply:
Huh. That's a fun one  . Yeah lemme fix that.